<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DateSelected</class>
 <widget class="QWidget" name="DateSelected">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="opponentTeamNameLabel">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>110</y>
     <width>231</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Opponent Team Name:</string>
   </property>
  </widget>
  <widget class="QLabel" name="dateLabel">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>70</y>
     <width>241</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Date:</string>
   </property>
  </widget>
  <widget class="QLabel" name="locationLabel">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>130</y>
     <width>221</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Location:</string>
   </property>
  </widget>
  <widget class="QLabel" name="timeLabel">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>90</y>
     <width>241</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Time:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="viewOurStartingButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>167</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="toolTip">
    <string>Click to view current starting players, update the starting players</string>
   </property>
   <property name="text">
    <string>View Our Starting XI</string>
   </property>
  </widget>
  <widget class="QPushButton" name="viewOpponentStartingButton">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>167</y>
     <width>191</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <bold>true</bold>
    </font>
   </property>
   <property name="toolTip">
    <string>Click to view oppenent starting players, update opponent starting players</string>
   </property>
   <property name="text">
    <string>View Opponent Starting XI</string>
   </property>
  </widget>
  <widget class="QPushButton" name="simulateMatchButton">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>230</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <bold>true</bold>
    </font>
   </property>
   <property name="toolTip">
    <string>Simulate match based on teams' statistics with results</string>
   </property>
   <property name="text">
    <string>Simulate Match</string>
   </property>
  </widget>
  <widget class="QLabel" name="infoLabel">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>270</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>17</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="toolTip">
    <string>Before simulating a match, make sure to choose a sufficient amount of players for both teams (exactly 11 players).</string>
   </property>
   <property name="text">
    <string>i</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
